modin/pandas/dataframe.py

```diff
@@ -2080,7 +2080,9 @@ def set_index(
             else:
                 frame = self.copy()
             if not all(
-                isinstance(col, (pandas.Index, Series, np.ndarray, list, Iterator))
+                isinstance(
+                    col, (pandas.Index, pandas.Series, np.ndarray, list, Iterator)
+                )
                 for col in keys
             ):
```

Comment:
this check should work the same however looks clearer IMO
```python
if any(isinstance(col, Series) for col in keys):
```

Reply:
or we can even drop this whole if-statement and replace it with a [`try_cast_to_pandas`](https://github.com/modin-project/modin/blob/21ab814e2f9fd9e4874a036e2fa8e53208638614/modin/utils.py#L480) call:
```python
keys = try_cast_to_pandas(keys)
```

Reply:
I switched to `try_cast_to_pandas`. Thanks for the suggestion.